q_sort.py

```diff
@@ -0,0 +1,60 @@
+from merge_sort import timed_func
+
+def q_sort(sequence): 
+    if len(sequence) <= 1:
+        # Return a sequence of one sorted item, or stop if no input
+        return sequence
+    if len(sequence) == 2:
+        # Return a sequence of two sorted items
+        if sequence[1] > sequence[0]:
+            return sequence
+        else:
+            return sequence[::-1]
+    pivot = get_pivot(sequence)
+    left = []
+    right = []
+    not_pivot = False
+    for item in sequence:
+
+        if item > pivot[0] and item is not pivot[0]:
```

Comment:
How could an item be simultaneously greater than another and exactly the same as another?  The second half of your predicate here makes no sense to me.